src/Layout.js

```diff
@@ -651,6 +673,11 @@ var computeLayout = (function() {
       }
     }
   };
+
+  return function performLayout(node, parentMaxWidth) {
+    prepareNodes(node);
```

Comment:
It's going to be slow to do that during each layout, but since we don't have caching yet on the JS version it's probably fine for now :)